.github/workflows/tests_unit.yml

```
@@ -92,7 +92,9 @@ jobs:
           arguments: jacocoUnitTestReport --daemon
 
       - name: Store Logcat as Artifact
-        if: failure()
+        # cancelled() handles test timeouts
+        # remove when test timeouts cause a failure()
+        if: failure() || cancelled()
```

It appears it *may* be (untested) possible to do:

```suggestion
        if: '!success()'
```

...and catch every single case except success, but failure && cancelled also seems sufficient